test/src/094-fuser/main

```diff
@@ -0,0 +1,45 @@
+
+cvmfs_test_name="Using cvmfs_config fuser to find processes accessing a repository"
+cvmfs_test_suites="quick"
+
+cvmfs_run_test() {
+  logfile=$1
+
+  echo "Mount repository"
+  cvmfs_mount grid.cern.ch || return 1
+
+  echo "Start background process accessing it and check that it is being used"
+  set -m # enables job control handling
+  (cd /cvmfs/grid.cern.ch; sleep 30) &
+  backpid="$!"
+  pids="`sudo cvmfs_config fuser grid.cern.ch`"
+  [[ " $pids " == *" $backpid "* ]] || return 10
+  kill -15 -$backpid
+  wait $backpid 2>/dev/null || true
+  pids="`sudo cvmfs_config fuser grid.cern.ch`"
+  [ -z "$pids" ] || return 11
+
+  echo "Start unshared background process accessing it, and check that"
+  unshare -Um sh -c "cd /cvmfs/grid.cern.ch && /bin/sleep 60" & 
+  backpid="$!"
+  sleep 1
```

Comment:
Is this necessary to wait for the unshared process to start?  If so, perhaps the unshared process could create a file and the main test waits for the file to appear. The sleeps have a tendency to fail every now and then (test VMs might be paused).